### PR TITLE
Adds Drillfall Dropper to the QM locker

### PR DESCRIPTION
**Pull request by Geevies**
* Added Drillfall, the ability to paint a target and drop two drills.
* Added the Drillfall Dropper to the QM locker.

### CODE REVIEW

**Comment by Geevies** [comment]
From a non-user facing standpoint, I also added a datum for a drilltech outfit. Currently it doesn't change much, but I want to work with Amory in the future to make a special armband or something that they wear.

**Comment by Geevies** [comment]
!review

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -184,6 +186,20 @@
 	dufflebag = /obj/item/weapon/storage/backpack/duffel/eng
 	messengerbag = /obj/item/weapon/storage/backpack/messenger/engi
 
+/datum/outfit/job/miningd
```
Comment (Alberyk) [inline code/game/jobs/job/civilian.dm]: Can't this just be a subtype of the mining outfit to avoid all this copypasta?

**Comment by Geevies** [comment]
@Alberyk yes

**Comment by fernerr** [comment]
I feel like there's not enough depth behind industrial drills mechanically to justify adding an alt-title just for them. I'd only be in favour of adding such a job if automated mining possibilities were expanded upon. 
Adding those drills at roundstart would also overpower mining even more, making them able to produce large amounts of resources even earlier on in the round, as nothing is hindering a single regular miner just setting up those drills in an instant and just heading out on their usual mining-run.

**Comment by Geevies** [comment]
I had similar concerns, the automation of drills would hopefully be something that can be accomplished by the construction update the other devs are working on. In that case, I'd include a conveyer construction device to the drilling kit.
One idea I have to not have miners just set it up in addition to them going out to mine, is to add the drills to a drilling outpost, which only Drill Technicians have access to. What's your thoughts on that?

**Comment by furrycactus** [comment]
Locking them somewhere with access that only Drill Technicians have would be a good start, at least imo.

**Comment by Geevies** [comment]
Alright, so instead of. Uh. Mapping in drills and lame ass storage sheds, I added an item that spawns in Drill Technicians' drill kits that allow them to drop drills into location. Each of these devices have two uses, and can serve as binoculars.

**Comment by Geevies** [comment]
Adding WIP; Drill mechanics are currently very boring (heh heh heh). I want to expand on them as part of this PR, make it actually fun.

**Comment by Geevies** [comment]
I changed this PR to not add the drill technician, instead adding just the drillfall dropper and adding it to the QM locker.

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/devices/drilldroptargetter.dm [hunk 2]
```diff
@@ -0,0 +1,117 @@
+/obj/item/device/drill_dropper
+	name = "laser targeting dropper"
+	desc = "Your drill shall pierce the heavens."
```
Comment (JohnWildkins) [inline code/game/objects/items/devices/drilldroptargetter.dm]: this reference is so overused on drills that thermal drills already make fun of this reference

Comment (JohnWildkins) [reply code/game/objects/items/devices/drilldroptargetter.dm]: honestly even just going with "This drill literally pierces the heavens." would be an improvement

Comment (Geevies) [inline code/game/objects/items/devices/drilldroptargetter.dm]:
```suggestion
	desc = "This drill literally pierces the heavens."
```

Comment (Geevies) [reply code/game/objects/items/devices/drilldroptargetter.dm]: The sad cowboy is right.

File: code/game/objects/items/devices/drilldroptargetter.dm [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+				return
+			if (!isfloor(targloc))
+				to_chat(user, "<span class='warning'>You cannot request a drill on unstable flooring!</span>")
+				return
+		for(var/turf/t2 in block(locate(targloc.x+2,targloc.y+2,targloc.z+1),locate(targloc.x-2,targloc.y-2,targloc.z+1)))
+			if (!istype(t2.loc, /area/mine))
+				to_chat(user, "<span class='warning'>You can't do this beneath the station, point the laser further away from it!</span>")
+				return
+	if(!(user in (viewers(14,target))) )
+		return
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/drilldroptargetter.dm]: Is it intended for the entire proc to not work if the user's not within 14 feet of the target? If so, it sounds like you might want a to_chat here too.

Comment (Geevies) [reply code/game/objects/items/devices/drilldroptargetter.dm]: Good call.

Comment (Geevies) [inline code/game/objects/items/devices/drilldroptargetter.dm]:
```suggestion
		to_chat(user, "<span class='warning'>You can't paint the target that far away!</span>")
		return
```

File: html/changelogs/geeves-drillTechandDrillTechAccessories.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a drill dropper item. It drops drills. Can't be used near station areas, unless tampered with.."
```
Comment (Arrow768) [inline html/changelogs/geeves-drillTechandDrillTechAccessories.yml]:
```suggestion
  - rscadd: "Added a drill dropper item. It drops drills. Can't be used near station areas, unless tampered with."
```

**Pull request merged**